test/index.js

```diff
@@ -78,10 +79,17 @@ describe("Test for infinity menu", function() {
 			onLeafMouseUp={onLeafMouseUpStub}
 			onLeafMouseDown={onLeafMouseDownStub}
 			onClose={onCloseStub}
+			headerMetadata={headerMetadata}
 			/>;
 		dom = TestUtils.renderIntoDocument(component);
 	});
 
+	it("should render correctly with header metadata", function () {
```

Comment:
I agree with @conorhastings feedback here. The test needs to validate the generated markup from setting the `headerMetadata` prop